witchcraft-server-ete/tests/ete/server/cert.cer

```diff
@@ -1,21 +1,18 @@
 -----BEGIN CERTIFICATE-----
```

Comment:
Need a new test cert (copied from conjure-rust-runtime) since rustls doesn't allow CAs to be used as leafs. This is a self-signed non-CA cert.